index.html

```diff
@@ -4297,19 +4304,60 @@ <h2>JSON Processing</h2>
-processors will use the specific mechanism provided and can verify that all
-terms are correctly specified, JSON-based processors implicitly accept the same
-semantics without performing any JSON-LD transformations, but instead by
+These rules alone do not guarantee semantic interoperability between JSON and JSON-LD for
+literal JSON keys mapped to URIs by the `@context` mechanism.
+          </p>
+While some RDF processors will use the language features of JSON-LD and can verify that all
+terms are correctly specified, JSON-LD processors explicitly accept the same
+semantics without performing any RDF transformations, but instead by
 applying the above rules. In other words, the context in which the data exchange
-happens is explicitly stated for both JSON and JSON-LD by using the same
-mechanism. With respect to JSON-based processors, this is achieved in a
-lightweight manner, without having to use JSON-LD processing libraries.
+happens is explicitly stated for both RDF and JSON-LD by using the same
+mechanism. With respect to JSON-LD-based processors, this is achieved in a
+lightweight manner, without having to use RDF processing libraries.
+          </p>
+        </section>
+        <section>
+          <h2>Advanced RDF Processing</h2>
+          <p>
+Issuers should be aware that while conforming documents are expressed as compact JSON-LD,
+not all Holders or Verifiers will understand expressions of intention that are only visible 
+after RDF processing has occured.
+          </p>
+          <p>
+Some of the best features of JSON-LD are only possible by applying its language features to the 
+terms defined in the <code>@context</code>.
+          </p>
+          <p>
+Implementers should be aware that <code>id</code> and <code>type</code> have been
+aliased to <code>@id</code> and <code>@type</code>, and are marked <code>@protected</code>.
+Depending on the software libraries used, attempts to redefine these terms may raise processing errors.
+Implementers interested in understanding what <code>@id</code> means 
+should review <a href="https://www.w3.org/TR/json-ld11/#node-identifiers">Node Identifiers</a>, and
+<a href="https://www.w3.org/TR/json-ld11/#specifying-the-type">Specifying the Type</a>.
+          </p>
+          <p>
+Implementers should be aware that object member, and array element order are not preserved by default in JSON-LD,
+and as such, Issuers intending to communicate array order need to leverage the langauge features of JSON-LD.
+See <a href="https://www.w3.org/TR/json-ld11/#sets-and-lists">Value Ordering</a>.
+          </p>
+          <p>
+Implementers should be aware that while naive JSON interpretation of claims might make it appear that claims are related,
+the <code>https://www.w3.org/ns/credentials/v2</code> leverages <code>@container</code> and <code>@graph</code> to separate information graphs.
+In particular, to separate the graphs related to <a>verifiable credentials</a>, <a>verifiable presentations</a>
```

Comment:
AFAIK there is no property marked as `"@container":"@graph"` for VP-s. Only credentials and proofs are "separated" in their own graphs.

Reply:
https://github.com/w3c/vc-data-model/blob/main/contexts/credentials/v2#L175

^ This one... and 

https://github.com/w3c/vc-data-model/blob/main/contexts/credentials/v2#L139

Is what I was referring too.

Reply:
What those mean is that a Verifiable Credential, resp. a proof, is put into a separate graph. You are right that this means, in practice, that the Verifiable Presentation that refers to the credential is therefore kept in its own (default) graph, but the sentence is nevertheless misleading.

Reply:
Its more misleading to ignore that RDF transformations make this distinction but JSON-LD (Without context processing) does not... how do you suggest we proceed?

Reply:
Remove the reference to verifiable presentation. References to proofs and to verifiable credentials result in separate graphs, as you say in the text, and that is it.

Reply:
@OR13, thoughts?

Reply:
I don't think this is thread is actionable.

The context controls the shape of the RDF, not processing it (the latest version published by the authority, such as W3C) will lead to inconsistent processing.

I suggest we resolve this thread, or file seperate issues for follow up

Reply:
> The context controls the shape of the RDF, not processing it (the latest version published by the authority, such as W3C) will lead to inconsistent processing.

I do not understand this statement.